src/app/settings/index.component.ts

```diff
@@ -57,12 +57,14 @@ import { StorageService } from '@services/storage.service';
           </mat-option>
         </mat-select>
       </mat-form-field>
-
+      
```

Comment:
Be careful with formating. At the beginning, you think that it's nothing and at the end, your code is just a soup of space, tab and useless new line which make the code harder to read and more conflits prone.